src/main/java/it/pagopa/transactions/configurations/CheckoutRedirectPspApiKeysConfig.java

```diff
@@ -0,0 +1,89 @@
+package it.pagopa.transactions.configurations;
+
+import com.fasterxml.jackson.core.JacksonException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.opentelemetry.api.trace.Span;
+import io.vavr.control.Either;
+import it.pagopa.transactions.exceptions.CheckoutRedirectConfigurationException;
+import it.pagopa.transactions.exceptions.CheckoutRedirectMissingPspRequestedException;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.*;
+
+/**
+ * This class take cares of parsing PSP api keys configuration for Checkout
+ * Redirect
+ */
+@Slf4j
+public class CheckoutRedirectPspApiKeysConfig {
```

Comment:
What about move this logic to commons library? This can be used in future by event dispatcher for retrieve api key for refund operation. This way we will avoid a future refactoring

Reply:
Mving this to commons